airbyte-integrations/connectors/source-google-search-console/source_google_search_console/streams.py

```diff
@@ -96,6 +97,7 @@ class Sitemaps(GoogleSearchConsole):
     API docs: https://developers.google.com/webmaster-tools/search-console-api-original/v3/sitemaps
     """
 
+    primary_key = ["type", "path", "lastSubmitted", "lastDownloaded"]
```

Comment:
it look like this PK is not unique according to expected records - https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-search-console/integration_tests/expected_records.jsonl#L3

Reply:
We have the `sitemaps` ignored fields like:
```yaml
  ignored_fields:
       sitemaps:
            - name: lastDownloaded
              bypass_reason: "URL changes upon each request for privacy/security"
            - name: contents/0/submitted
              bypass_reason: "URL changes upon each request for privacy/security"
``` 
ideally, the `path` field should hold 1 unique URL, which would be the PK essentially, but there is no guarantee it will work as expected. Should I leave this stream with no PK? `Sitemaps` is the `full-refresh` stream only, FYI.

Reply:
@lazebnyi Updated the PR by removing the PK for `Sites` and `Sitemaps`, they are `full-refresh` streams with no option for dedupe.